src/content/app/tools/blast/components/blast-app-bar/BlastAppBar.tsx

```diff
@@ -48,6 +49,14 @@ const BlastAppBar = (props: Props) => {
   const speciesListIds = useSelector(getSelectedSpeciesIds);
   const dispatch = useDispatch();
 
+  const speciesSelectorLink = useMemo(() => {
+    return <Link to={urlFor.speciesSelector()}>Change</Link>;
+  }, []);
```

Comment:
The reason the memo was there is that the components are then passed as a prop to another component, and without the memo it would rerender.

https://github.com/Ensembl/ensembl-client/blob/92330b1d4abc8b77636bf43f3c90fd8205ff4c0f/src/content/app/tools/blast/components/blast-app-bar/BlastAppBar.tsx#L84-L88

One can argue that we don't especially care anyway, since we are passing non-memoized functions left and right. But then, on the other hand, if we can prevent unnecessary rerenders, then why won't we?

Reply:
Now it sounds like memoizing is better? so whats the decision?

Reply:
I didn't know about the re-rendering issue here. In that case it would be better to memoize this again.